sound/soc/sof/intel/hda.c

```diff
@@ -618,8 +618,8 @@ int hda_dsp_probe(struct snd_sof_dev *sdev)
 	}
 
 	dev_dbg(sdev->dev, "using IPC IRQ %d\n", sdev->ipc_irq);
-	ret = request_threaded_irq(sdev->ipc_irq, hda_dsp_ipc_irq_handler,
-				   sof_ops(sdev)->irq_thread, IRQF_SHARED,
+	ret = request_threaded_irq(sdev->ipc_irq, NULL,
+				   hda_dsp_ipc_irq_handler, IRQF_SHARED,
```

Comment:
sorry I don't get this one. this would mean we use a generic handler and the current handler becomes a thread?
Is this needed?

Reply:
Yeah, I thought there are no other interrupt share the irq and want to do a quick test yesterday. Now I change it back as it is.